.github/workflows/create_release.yml

```diff
@@ -0,0 +1,144 @@
+name: "Create release"
+
+on:
+  pull_request:
+    types:
+      - closed
+
+env:
+  FLUTTER_VERSION: "3.7.7"
+  RUST_VERSION: "1.68.0"
+
+jobs:
+  build:
+    runs-on: macos-latest
+    if: github.event.pull_request.merged == true && startsWith(github.event.pull_request.head.ref, 'release/')
+    outputs:
+      ANDROID_APK_NAME: ${{ steps.build.outputs.ANDROID_APK_NAME }}
+    steps:
+      - uses: actions/checkout@v2
+
+      # #499, https://github.com/actions/virtual-environments/issues/5595
+      - name: Configure ndk
+        run: |
+          ANDROID_HOME=$HOME/Library/Android/sdk
+          SDKMANAGER=$ANDROID_HOME/cmdline-tools/latest/bin/sdkmanager
+
+          echo y | $SDKMANAGER "ndk;21.4.7075529"
+
+          ln -sfn $ANDROID_HOME/ndk/21.4.7075529 $ANDROID_HOME/ndk-bundle
+
+      - name: Setup | Rust
+        uses: ATiltedTree/setup-rust@v1
+        with:
+          rust-version: ${{ env.RUST_VERSION }}
+          components: rustfmt
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: "temurin"
+          java-version: "11.x" # "betterprogramming.pub" says must be java "8"
```

Comment:
the comment says it should be java 8 but here it is java 11?

Reply:
removed the comment 😝